src/libraries/Microsoft.Extensions.Hosting/tests/TestApp/Microsoft.Extensions.Hosting.TestApp.csproj

```diff
@@ -1,35 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
-
   <PropertyGroup>
-    <TargetFramework>$(DefaultNetCoreTargetFramework)</TargetFramework>
-    <OutputType>Exe</OutputType>
+    <TargetFramework>$(NetFrameworkCurrent)</TargetFramework>
+    <EnableDefaultItems>true</EnableDefaultItems>
+    <TargetExt>.exe</TargetExt>
```

Comment:
Let's just keep `<OutputType>Exe</OutputType>`.  The other projects we were looking at used TargetExt so that they could share the same code between desktop and core.  https://github.com/dotnet/runtime/blob/efe347f9dd5820aaf8504f9c69f099d2179141ec/src/libraries/System.Threading.Thread/tests/STAMain/STAMain.csproj#L4